Assets/Scripts/Models/Furniture.cs

```diff
@@ -766,12 +767,16 @@ public IEnumerable<ContextMenuAction> GetContextMenuActions(ContextMenu contextM
 
         foreach (ContextMenuLuaAction contextMenuLuaAction in contextMenuLuaActions)
         {
-            yield return new ContextMenuAction
+            if (!contextMenuLuaAction.DevModeOnly ||
+                Settings.GetSettingAsBool("DialogBoxSettings_developerModeToggle", false))
```

Comment:
This is a compile error now. The new prototype is just `GetSetting` and it infers the type form the default, so you can remove `AsBool`.